vtigercron.php

```diff
@@ -57,10 +61,12 @@
 					}
 				}
 				if (!$run_task) {
-					$msg = sprintf("[INFO]: %s - not ready to run as the time to run again is not completed\n", $cronTask->getName());
-					echo $msg;
-					$logbg->info($msg);
-					continue;
+					if (empty($argv[1]) || $argv[1] != '--quiet' || $argv[1] != '-q') {
```

Comment:
this condition is incorrect, I think you need parenthesis around the options, think about it and get back to me.